server/views/pages/work.njk

```diff
@@ -1,6 +1,8 @@
 {% extends 'layout/default.njk' %}
 
-{% block pageMeta %}{% endblock %}
+{% block pageMeta %}
+  <link rel="canonical" href="{{ work.requestOrigin }}{{ work.requestPath }}" />
```

Comment:
TO be a complete pedant - we could just `canonicalUri` on the root of the `pageConfig` URL, just a thought as it seems page specific.